src/engine/blkalloc/blk_allocator.cpp

```diff
@@ -28,6 +28,13 @@ BlkAllocator::BlkAllocator(const BlkAllocConfig& cfg, const chunk_num_t id) : m_
     const auto bitmap_id{id};
     m_disk_bm = std::make_unique< sisl::Bitset >(m_cfg.get_total_blks(), bitmap_id, align_size);
 
+#if defined _PRERELEASE || defined DEBUG
```

Comment:
I think we should move this logic inside BlkAllocatorConfig::BlkAllocatorConfig so that there is only one place that sets the realtime bitmap. For 1.2 we can just leave as it is.

Reply:
Yes. I believe your suggestion makes more sense than this one which is identical to 1.2!